source/api/discovery/0.5/index.md

```diff
@@ -279,15 +290,15 @@ As the number of Activities is likely too many to usefully be represented in a s
 
 The W3C [Activity Streams][org-w3c-activitystreams] specification defines a "model for representing potential and completed activities", and is compatible with the [design patterns][annex-patterns] established for IIIF APIs. It is defined in terms of [JSON-LD][org-w3c-json-ld] and can be seamlessly integrated with the existing IIIF APIs. The model can be used to represent activities carried out by content publishers of creating, updating, and deleting (or otherwise de-publishing) IIIF resources.
 
-This section is a summary of the properties and types used by this specification, and defined by Activity Streams.  This is intended to ease implementation efforts by collecting the relevant information together.
```

Comment:
I see you sneaking in double spaces after stops. Don't think I won't catch you ;-)

Reply:
:D Fingers are programmed for 30+ years to add double spaces. Hard to revert!